apps/studio/prisma/constants.ts

```diff
@@ -25,7 +22,13 @@ export const ISOMER_MIGRATORS = [
   "felicia",
 ]
 
-export const PAST_ISOMER_MEMBERS = ["kishore", "hanpu"] as const
+export const PAST_ISOMER_MEMBERS = [
```

Comment:
saw that it's used to determine user deletion, but do we actually need to maintain this list?

Reply:
It's used to display users under the Isomer Admin table in User management instead of agency users since we don't have a super admin role.

We won't need it if we remove them when they leave, but given that they are likely added to all sites, it's quite a hassle to manually remove them from all sites - will let the Cron job auto removed them after 90 days